airflow/logging_config.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import logging
+import os
+import sys
+
+from logging.config import dictConfig
+from airflow import configuration as conf
+
+log = logging.getLogger(__name__)
+
+
+def configure_logging(custom_config_path=None):
+    if custom_config_path is not None:
+        config_path = custom_config_path
+    else:
+        config_path = conf.get('core', 'logging_config_path')
+
+    try:
+        path = os.path.dirname(config_path)
```

Comment:
The default for logging_config_path is an empty string, which I think causes the "Unable to load custom logging, using default config instead" message to be printed (I think)

It would be better to print nothing in the case of the default setting and just use the stock config without logging about it.

Reply:
What about setting the logging level to debug for this?